webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/activity/header/ActivityHeadTime.vue

```diff
@@ -25,14 +25,14 @@
             v-if="isActivityEdited"
             :value="activity.updateDate"
             :short="isMobile"
-            :class="truncateText"
+            :class="[truncateText, textNone]"
```

Comment:
couldn't we just remove the class **text-capitalize-first-letter** and rely on Crowdin texts ?